pkg/logging/logfields/logfields.go

```diff
@@ -93,8 +93,10 @@ const (
 	// IngressClass is the identifier of an ingress class object
 	IngressClass = "ingressClass"
 
-	// OldIdentity is a previously used security identity
-	OldIdentity = "oldIdentity"
+	// IdentityOld is a previously used security identity
+	IdentityOld = Identity + ".old"
```

Comment:
What about `IdentityOld = "old-" + Identity`? Seems clearer to me (same for `IdentityNew` and `CEPUIDOld`)